MonMaraicher/Model/Farmer/Farmers.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FarmersPlaces.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 13/03/2024.
+//
+
+import Foundation
+
+struct Farmer: Decodable, Identifiable, Hashable {
+    let id: Int
+    let name: String
+    let location: Location
+    let images: FarmerImages
+}
+
+struct FarmerImages: Decodable, Hashable {
+    let farmer1, farmer2, farmer3, farmer4, farmer5, farmer6: String
```

Comment:
on evite cette ecriture si possible, apres je crois que c'est pour depanner car tu veux eviter d'ecrire cette objet non ?

Reply:
Yes, I have no idea how I name the different images of the farmer

Reply:
Nan je parle de déclarer plusieurs property sur la même ligne. C'est vraiment à éviter.

Reply:
FIx